EIPS/eip-6551.md

```diff
@@ -0,0 +1,393 @@
+---
+eip: 6551
+title: Non-fungible Token Bound Accounts
+description: An interface and registry for smart contract accounts owned by ERC-721 tokens
+author: Jayden Windle (@jaydenwindle), Benny Giang <bg@futureprimitive.xyz>, Steve Jang, Druzy Downs (@druzydowns), Raymond Huynh (@huynhr), Alanah Lam <alanah@futureprimitive.xyz>
+discussions-to: https://ethereum-magicians.org/t/non-fungible-token-bound-accounts/13030
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-02-23
+requires: 155, 165, 721, 1167, 1271
+---
+
+## Abstract
+
+This proposal defines a system which gives every [ERC-721](./eip-721.md) token a smart contract account. These token bound accounts allow ERC-721 tokens to own assets and interact with applications, without requiring changes to existing ERC-721 smart contracts or infrastructure.
+
+## Motivation
+
+The ERC-721 standard enabled an explosion of non-fungible token applications. Some notable use cases have included breedable cats, generative artwork, and liquidity positions.
+
+Non-fungible tokens are increasingly becoming a form of on-chain identity. This follows quite naturally from the ERC-721 specification - each non-fungible token has a globally unique identifier, and by extension, a unique identity.
+
+Unlike other forms of on-chain identity, ERC-721 tokens cannot act as an agent or associate with other on-chain assets. This limitation stands in contrast with many real-world instances of non-fungible assets. For example:
+
+- A character in a role-playing game that accumulates assets and abilities over time based on actions they have taken
+- An automobile composed of many fungible and non-fungible components
+- An automated investment portfolio composed of multiple fungible assets
+- A punch pass membership card granting access to an establishment and recording a history of past interactions
+
+Several proposals have attempted to give ERC-721 tokens the ability to own assets. Some notable proposals include [ERC-998](./eip-998.md) and [ERC-6150](./eip-6150.md).
```

Comment:
I would very much recommend removing these references. Your proposal will not be able to advance past `Draft` unless all proposals linked also advance past `Draft`.

```suggestion
Several proposals have attempted to give ERC-721 tokens the ability to own assets.
```

Reply:
Ah, didn't realize that. Will remove!